docs/developer-guide/working-with-plugins.md

```diff
@@ -24,6 +24,22 @@ module.exports = {
 
 To use the rule in ESLint, you would use the unprefixed plugin name, followed by a slash, followed by the rule name. So if this plugin were named `eslint-plugin-myplugin`, then in your configuration you'd refer to the rule by the name `myplugin/dollar-sign`. Example: `"rules": {"myplugin/dollar-sign": 2}`.
 
+#### Linking to Rule Documentation
+
+If your plugin has rules, you can tell ESLint where to find the documentation for those rules. The
+URI will be passed on via the API so that editors and other clients can take advantage of it.
+
+To specify where the documentation will be found, include a getDocURI function.
+
+```js
+module.exports = {
+    getDocURI: function(ruleName) {
+        return "http://path.to/docs/" + ruleName;
+    }
```

Comment:
Thinking it may be useful to pass the problem object as a second argument so the plugin can link to more specific detail about how the rule was violated. An IDE could even fetch the URL and provide contextual help. 

But that's another PR for another day.